scss/_nav.scss

```diff
@@ -26,18 +26,22 @@
   }
 }
 
-
 //
 // Tabs
 //
 
 .nav-tabs {
   border-bottom: $nav-tabs-border-width solid $nav-tabs-border-color;
 
-  .nav-item {
+  .nav-item,
+  .nav-link {
```

Comment:
Or we can require `nav-item` the same you did for `nav-fill` and `nav-justified` ?
The markup would be like:
```html
<ul class="nav nav-tabs">
  <li class="nav-item">
    <a class="nav-link" href="#">Item</a>
  </li>
</ul>
```
or
```html
<nav class="nav nav-tabs">
    <a class="nav-item nav-link" href="#">Item</a>
</nav>
```